### PR TITLE
add missing types

### PR DESCRIPTION
**Pull request by Kachulio1**
## Overview

Brief description of what this PR does, and why it is needed.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected output

### CODE REVIEW

**Comment by jobala** [comment]
LGTM.

File: src/app/views/authentication/profile/Profile.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import { IProfileProps } from '../../../../types/profile';
```
Comment (jobala) [inline src/app/views/authentication/profile/Profile.tsx]: Add a new line here.  

We separate node_module and local imports with a new line

File: src/app/views/query-runner/request/Request.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { Pivot, PivotItem, PivotLinkSize } from 'office-ui-fabric-react';
 import React, { Component } from 'react';
+import { IRequestState } from '../../../../types/request';
```
Comment (jobala) [inline src/app/views/query-runner/request/Request.tsx]: Here too

File: src/app/views/authentication/profile/Profile.tsx (hunk 1 above)
Comment (Kachulio1) [reply src/app/views/authentication/profile/Profile.tsx]: 👍

**Pull request merged**